src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -84,9 +84,6 @@ private void handleChildSupportAndSscs5Case(Callback<SscsCaseData> callback) {
                 if (isBenefitTypeSscs5(callback.getCaseDetails().getCaseData().getBenefitType())) {
                     updateEventDetails(caseData, callback.getCaseDetails().getId(),
                         EventType.DWP_RESPOND, "Response received", "Update to response received as an Admin has to review the case");
-                } else {
```

Comment:
Just checking, by removing this, does that mean SSCS1/2 cases will have event details updated to "Response received" somewhere else? Based on AC that's what should be set (at least for child support cases).

Reply:
Prior to running "Upload response" event, the case state is already at "Response received" for all case types afaik. The purpose of this ticket is to not update it to "Not listable" for sscs2 cases, but can be updated for other cases which are handled by handleUc() and handleNonUc() methods. Does this answer your question?

Reply:
Cool, good to know.

Reply:
Actually, what happens is that when you submit upload response on any case, it goes to "Response received". Then the case state is updated in this handler. See the End state of the case 1681899688180933 in aat environment when upload response is submitted. Then its send to not listable because of this code.